openapi.yaml

```diff
@@ -6786,14 +6767,48 @@ components:
     Bearer:
       type: http
       scheme: bearer
-      bearerFormat: >-
-        The Bearer Token MUST consist of the authentication method, a provider
-        ID (if available) and the token itself. All separated by a forward slash
-        `/`. Examples (replace `TOKEN` with the actual access token): (1) Basic
-        authentication (no provider ID available): `basic//TOKEN` (2) OpenID
-        Connect (provider ID is `ms`): `oidc/ms/TOKEN`. For OpenID Connect, the
-        provider ID corresponds to the value specified for `id` for each
-        provider in `GET /credentials/oidc`.
+      bearerFormat: JWT or openEO
+      description: >-
+        A Bearer token can be provided in two different formats:
+
+        1. **JSON Web Token (JWT) - RECOMMENDED**
+
+           - Conformance class: `https://api.openeo.org/1.2.0/authentication/jwt`
+           
+           The Bearer token is an access token in [JWT](https://datatracker.ietf.org/doc/html/rfc7519) format
+           as defined in RFC 7519. For openEO, it MUST include the issuer in the
+           `iss` claim although being optional in RFC 7519.
+           If the concept of an issuer doesn not exist in an authentication method (e.g. in HTTP Basic),
+           implementations could use the endpoint for Basic Authentication as the issuer, for example.
+
+           If this method is supported by the openEO implementation,
+           the server MUST provide the conformance class given above.
+
+        2. **openEO Tokens - DEPRECATED**
+
+           - Conformance class: *None*
+
+           The Bearer Token is constructed from the authentication method, a
+           provider ID (if available) and the access token. All separated by a
+           forward slash `/`.
+
+           Examples (replace `TOKEN` with the actual access token):
+
+           - Basic authentication (no provider ID available): `basic//TOKEN`
+           - OpenID Connect (provider ID is `ms`): `oidc/ms/TOKEN`.
+             For OpenID Connect, the provider ID corresponds to the value
+             specified for `id` for each provider in `GET /credentials/oidc`.
+
+           All servers must accept this method for backward compatibility
+           until version 2.0 of the specification.
+
+          The access tokens provided by the identity provider do not include
```

Comment:
I think there is something off identation-wise here. Selectively picking particular lines to highlight:

```
        2. **openEO Tokens - DEPRECATED**
           All servers must accept this method for backward compatibility
          The access tokens provided by the identity provider do not include
```

Reply:
Thanks for spotting this, fixed.